content/java/concepts/math-methods/terms/floor/floor.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'Floor'
+Description: 'Returns the largest integer value that is less than or equal to the argument. When the provided value is either an integer, zero, not a number, or positive or negative infinity.'
+Subjects: 
+    -'Code Foundations'
+    -'Computer Science'
+Tags: 
+    - 'Documentation'
+CatalogContent: 
+  - 'learn-java/static-methods-of-the-math-class'
+---
+
+The static function **`.floor()`** included in the `Math` class returns the largest integer value that is less than or equal to the argument. When the provided value is either an integer, zero, not a number, or positive or negative infinity.
+
+## Syntax
+The Math package can be imported and utilized as shown below
+```java
+    import java.lang.Math;
+
+    double x = 1.1;
+    Math.floor(1.1); // passing values directly to the function
+    Math.floor(x); // or passing values of variables
```

Comment:
```suggestion
Math.floor(x)
```

For the code block in the "Syntax" section, let's try to keep it to 1-2 lines, in most cases. This should be a place to display the general structure of the concept/term being presented.

Reply:
Suggestion implemented.